ipam/client/abstractipam.py

```diff
@@ -18,6 +18,10 @@ def add_next_ip(self, subnet, dnsname, description):
     def get_next_free_ip(self, subnet):
         raise NotImplementedError()
 
+    @abstractmethod
+    def add_master_subnet(self, subnet, description):
```

Comment:
master_subnet is a bit confusing. What about add_top_level_subnet (or anything better if you have another suggestion)?